2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,60 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+
+while (list.hasChildnodes()) {
+  list.removeChild(list.lastChild);
+}
+todos.forEach(todo)=> {
```

Comment:
There's one missing ( here. Code looks really nice otherwise, with descriptive varible names in quite a few places and you using a full range of methods, and the result looks good in the browser  but sadly doesn't actually work